opencl-aot/include/utils.h

```diff
@@ -14,6 +14,13 @@
 #ifndef AOT_COMP_TOOL_UTILS
 #define AOT_COMP_TOOL_UTILS
 
+// Suppress a compiler warning about undefined CL_TARGET_OPENCL_VERSION
+// and define all symbols up to OpenCL 2.2
+#ifndef CL_TARGET_OPENCL_VERSION
+#define CL_TARGET_OPENCL_VERSION 220
+#endif
```

Comment:
AFAIK, we don't expose openc-aot is a library to the users, so the problem we discussed for the sycl library doesn't apply here. Setting this macro in CMake script should be enough. Right?
Do you expect this header to be included by some other project, which doesn't necessary sets this macro?

Reply:
> Do you expect this header to be included by some other project, which doesn't necessary sets this macro?

Not really, I just added it for consistency with `sycl`, as it seemed harmless enough.

Reply:
Do you prefer that I remove it ?

Reply:
I don't care that much about this. This is very minor issue in my opinion.
I'll let @dm-vodopyanov decide as he owns this tool.

Thanks for addressing the rest of the comments!

Reply:
I would prefer to remove this. Setting CMake variable removes warnings from the build logs. It doesn't need to define marco there as we deliver opencl-aot as an executable only.

Reply:
Sure, no problem.